res/raw/webxdc_wrapper.html

```diff
@@ -129,8 +129,6 @@
           } else {
             loadingDiv.innerHTML = "";
             iframe.src = "index.html";
-            iframe.contentWindow.webxdc_internal = window.webxdc_internal;
-            iframe.contentWindow.webxdc = window.webxdc;
```

Comment:
in my understanding:

- `window.webxdc` in `webxdc_wrapper.html`  is not set at all, so `null`
- `window.webxdc` in `index.html` may be set when `webxdc.js` is included. from outside, this is reachable as `iframe.contentWindow.webxdc`

in theory, this lines could even overwrite the just loaded webxdc api in the iframe with `null` - probably this just does not happen as javascript is single threaded and `iframe.src` is executed later.

please correct me if i am wrong :)

Reply:
Makes sense. Perhaps it was just copied from the desktop version.
https://github.com/deltachat/deltachat-desktop/pull/3098/commits/16bdbaa01b634ef4d2bcd98dd86ef712baa157a8#diff-15491a0f9e706d644386a23d0996ebec3a6b94f82a25a220f409272a359fb0c0R149-R150